controllers/apps/operations/backup.go

```diff
@@ -165,6 +196,38 @@ func getDefaultBackupPolicy(reqCtx intctrlutil.RequestCtx, cli client.Client, cl
 	return defaultBackupPolices.Items[0].GetName(), nil
 }
 
+func getDefaultBackupMethod(reqCtx intctrlutil.RequestCtx, cli client.Client, cluster *appsv1alpha1.Cluster, backupPolicyName string, backupMethod string) (string, error) {
```

Comment:
this function has duplicated code with getBackupMethodsFromBackupPolicyTemplates in cli package